pkg/plugins/resources/npm/condition.go

```diff
@@ -0,0 +1,41 @@
+package npm
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/updatecli/updatecli/pkg/core/pipeline/scm"
+	"github.com/updatecli/updatecli/pkg/core/result"
+)
+
+// Condition checks that an Npm package version exist
+func (n Npm) Condition(source string) (bool, error) {
+	versionToCheck := n.spec.Version
+	if versionToCheck == "" {
+		versionToCheck = source
+	}
+	if len(versionToCheck) == 0 {
+		return false, errors.New("no version defined")
+	}
+
+	_, versions, err := n.getVersions()
+	if err != nil {
+		return false, err
+	}
+
+	for _, v := range versions {
+		if v == versionToCheck {
+			fmt.Printf("%s release version '%s' available\n", result.SUCCESS, versionToCheck)
+			return true, nil
+		}
+	}
+
+	fmt.Printf("%s Version '%v' doesn't exist\n", result.FAILURE, versionToCheck)
```

Comment:
```suggestion
	fmt.Printf("%s Version %q doesn't exist\n", result.FAILURE, versionToCheck)
```

%q used with a string will put quotes around